### PR TITLE
Readme for Weblate translations

### PR DESCRIPTION
**Pull request by dpanshug**
Adding a description of Weblate Translations and how to contribute.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,21 @@ Then `mvn clean install -De2e=true`
...
 
+## Translation
+
+We are using Webplate for translating the Infinispan Console into different languages. Webplate is a web-based translation platform that streamlines the translation process and allows collaboration among contributors.
+
+You can access the translation project for Infinispan Console on Webplate [here](https://hosted.weblate.org/projects/infinispan/infinispan-console/). If you're interested in contributing translations or improving existing ones, feel free to join the Webplate project and start translating.
+
+### How to Contribute Translations
+
+1. Visit the [Webplate project](https://hosted.weblate.org/projects/infinispan/infinispan-console/) for Infinispan Console.
```
Comment (tristantarrant) [inline README.md:105]: s/webplate/weblate/

File: README.md [hunk 2]
```diff
@@ -94,6 +94,21 @@ Then `mvn clean install -De2e=true`
 
 Generated with [Favicon generator](https://www.favicon-generator.org/)
 
+## Translation
+
+We are using Webplate for translating the Infinispan Console into different languages. Webplate is a web-based translation platform that streamlines the translation process and allows collaboration among contributors.
+
+You can access the translation project for Infinispan Console on Webplate [here](https://hosted.weblate.org/projects/infinispan/infinispan-console/). If you're interested in contributing translations or improving existing ones, feel free to join the Webplate project and start translating.
```
Comment (tristantarrant) [inline README.md:101]: s/webplate/weblate/

File: README.md [hunk 3]
```diff
@@ -94,6 +94,21 @@ Then `mvn clean install -De2e=true`
 
 Generated with [Favicon generator](https://www.favicon-generator.org/)
 
+## Translation
+
+We are using Webplate for translating the Infinispan Console into different languages. Webplate is a web-based translation platform that streamlines the translation process and allows collaboration among contributors.
```
Comment (tristantarrant) [inline README.md:99]: s/webplate/weblate/

**Review by tristantarrant — CHANGES_REQUESTED** [review]
:)

**Comment by tristantarrant** [comment]
You get the idea :)

File: README.md [hunk 4]
```diff
@@ -94,6 +94,21 @@ Then `mvn clean install -De2e=true`
...
+
+You can access the translation project for Infinispan Console on Webplate [here](https://hosted.weblate.org/projects/infinispan/infinispan-console/). If you're interested in contributing translations or improving existing ones, feel free to join the Webplate project and start translating.
+
+### How to Contribute Translations
+
+1. Visit the [Webplate project](https://hosted.weblate.org/projects/infinispan/infinispan-console/) for Infinispan Console.
+2. Sign up or log in to your Webplate account.
+3. Choose the language you want to work on from the available languages or add a new language for translation.
+4. Start translating the strings directly on the Webplate interface.
+5. Once you're satisfied with the translations, submit them, and they will be reviewed and integrated into the Infinispan Console.
```
Comment (dpanshug) [inline README.md:109]: @tristantarrant, I'm not really sure how the translations will be reviewed, pls review these steps.

Comment (tristantarrant) [reply README.md:109]: I think we can have it create a PR. I'll check the config

File: README.md (hunk 1 above)
Comment (dpanshug) [reply README.md:105]: Sorry, I did not understand your comment.

**Comment by dpanshug** [comment]
@tristantarrant pls see if we can merge this

**Pull request merged**